python/ml4ir/applications/classification/pipeline.py

```diff
@@ -168,6 +170,33 @@ def get_relevance_dataset(
 
         return relevance_dataset
 
+    def finish(self, job_status, job_info):
+        # This will write the _SUCCESS/_FAILURE files
+        super().finish(job_status, job_info)
+
+        # Create model_bundle.zip
+        if self.args.file_handler == FileHandlerKey.SPARK:
+            # Copy tfrecord files in model_files
+            os.makedirs("epmm")
```

Comment:
Let's name this something more generic?

Reply:
yes please.  Probably have it configured by the commandline. 

("epmm" means nothing in the open source / outside of Salesforce usage of this codebase)